portfolio/src/main/webapp/index.html

```diff
@@ -65,8 +65,8 @@ <h4>Personal Project</h4>
       <hr>
 
 
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <p>Click the button below to receive a warm greeting!</p>
```

Comment:
Sorry because I am not 100% sure of context here, but should this be changed to "send a warm greeting" ?

Reply:
In this context it would be "receive" because when the user clicks the button then a greeting message will display below it.